packages/terra-search-field/src/SearchField.jsx

```diff
@@ -306,8 +306,8 @@ class SearchField extends React.Component {
         && (
         <label className={cx('label')}>{groupName}</label>
         )}
-        <div role="search" className={cx('search-role-container')} aria-label={intl.formatMessage({ id: 'Terra.searchField.search' })}>
-          <div role="group" aria-label={groupNameValue} {...customProps} className={searchFieldClassNames}>
+        <div className={cx('search-role-container')}>
+          <div role="group" aria-label={intl.formatMessage({ id: 'Terra.searchField.search' })} {...customProps} className={searchFieldClassNames}>
```

Comment:
what will be the screen reader response when groupName is `Search`. ( since this div and input will have same value for aria-label attribute will it result in redundant response

Reply:
Yes, it's causing redundant response of "Search".

Reply:
then we should remove the default Label from input aria-label to avoid the redundant response